src/main/java/voidcat/task/Event.java

```diff
@@ -0,0 +1,42 @@
+package voidcat.task;
+
+import voidcat.task.Task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+    protected static final DateTimeFormatter FORMATTER_DATETIME = DateTimeFormatter.ofPattern("uuuu-MM-dd HHmm");
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = LocalDateTime.parse(from, FORMATTER_DATETIME);
+        this.to = LocalDateTime.parse(to, FORMATTER_DATETIME);
+    }
+
+    public Event(String description, String from, String to, int done) {
+        super(description);
+        this.from = LocalDateTime.parse(from, FORMATTER_DATETIME);
+        this.to = LocalDateTime.parse(to, FORMATTER_DATETIME);
```

Comment:
Instead of repeating the same code, you can just call the other constructor.
```suggestion
        this(description, by); // Call the other constructor
```